packages/salesforcedx-apex-debugger/src/core/breakpointService.ts

```diff
@@ -62,6 +62,16 @@ export class BreakpointService {
     return this.typerefMapping.get(typeref);
   }
 
+  public getSourcePathFromPartialTyperef(
```

Comment:
This method (and associated logic) looks like potential candidate for future refactoring/higher re-use outside of breakpoint service.